src/Core/Domain/Country/ValueObject/CountryId.php

```diff
@@ -64,7 +64,7 @@ public function getValue(): int
      */
     private function assertPositiveInt(int $value)
     {
-        if (0 > $value) {
+        if (0 >= $value) {
```

Comment:
Are you really sure about it ?

(And why not use the constant ?)

Reply:
Yes sure for the >= (it's the aim of this PR)
And yes for for the constant. I applied this.